File: SpEditText/src/main/java/com/sunhapper/x/spedit/view/KeyEventProxy.java
package com.sunhapper.x.spedit.view;

import android.text.Editable;
import android.view.KeyEvent;

/**
 * Created by sunhapper on 2019/1/25 .
 */
public interface KeyEventProxy {

    boolean onKeyEvent(KeyEvent keyEvent, Editable text);
}

File: SpEditText/src/main/java/com/sunhapper/x/spedit/view/KeyEventProxy.kt
package com.sunhapper.x.spedit.view

import android.text.Editable
import android.view.KeyEvent

/**
 * Created by sunhapper on 2019/1/25 .
 */
interface KeyEventProxy {

    fun onKeyEvent(keyEvent: KeyEvent, text: Editable): Boolean
}
